src/Avalonia.Themes.Fluent/TextBox.xaml

```diff
@@ -156,4 +166,25 @@
     <Setter Property="Cursor" Value="IBeam" />
```

Comment:
Cursor=IBeam should be set only on inner DockPanel (there are two of them in the template), so InnerLeft and InnerRight will have default cursor.

Reply:
Fixed in https://github.com/AvaloniaUI/Avalonia/pull/4487